test/e2e/e2e_suite_test.go

```diff
@@ -281,9 +341,51 @@ func initScheme() *runtime.Scheme {
 	sc := runtime.NewScheme()
 	framework.TryAddDefaultSchemes(sc)
 	_ = infrav1.AddToScheme(sc)
+	_ = vcsimv1.AddToScheme(sc)
 	return sc
 }
 
+func amendE2EConfig() {
```

Comment:
optional: we could integrate this into `vsphereframework.LoadE2EConfig`, but ok to keep as is :-)

Reply:
Let's keep this here for now, it depends on global variables